sktime/benchmarking/_lib_mini_kotsu/run.py

```diff
@@ -58,16 +59,22 @@ def run(
         results_df = pd.DataFrame(columns=["validation_id", "model_id", "runtime_secs"])
         results_df["runtime_secs"] = results_df["runtime_secs"].astype(int)
 
+    from tqdm import tqdm
+
     results_df = results_df.set_index(["validation_id", "model_id"], drop=False)
     results_list = []
 
-    for validation_spec in validation_registry.all():
+    print("Printing Validations...")
+    print("\n")
+    for validation_spec in tqdm(validation_registry.all()):
```

Comment:
I would propose to avoid the print statements, you can add a description of the loop into the tqdm call.